presto-mysql/src/test/java/io/prestosql/plugin/mysql/TestMySqlDistributedQueries.java

```diff
@@ -30,24 +31,21 @@
 public class TestMySqlDistributedQueries
         extends AbstractTestDistributedQueries
 {
-    private final TestingMySqlServer mysqlServer;
+    private TestingMySqlServer mysqlServer;
 
-    public TestMySqlDistributedQueries()
+    @Override
+    protected QueryRunner createQueryRunner()
             throws Exception
     {
-        this(new TestingMySqlServer("testuser", "testpass", "tpch"));
```

Comment:
shouldn't this jut be supplier?

Reply:
i don't understand. it couldn't have been